riscv-debug-spec.tex

```diff
@@ -400,7 +400,8 @@ \subsection{Reset Control} \label{reset}
 programs from the first instruction executed, so exactly what is affected
 by this reset is implementation dependent. Debug Module's own state and registers are
 generally reset at power-up and if and only if
-\Fdmactive in \Rdmcontrol is 0.
+\Fdmactive in \Rdmcontrol is 0. This means that the halt state of all harts is
+maintained provided that \Fdmactive is 1, although breakpoints may be cleared.
```

Comment:
Suggest you call this "triggers" or "trigger CSRs". "Breakpoints" isn't really a term used in the spec, and that makes it more clear.